assets/src/components/v2/disruption_diagram/disruption_diagram.tsx

```diff
@@ -706,6 +562,7 @@ const DisruptionDiagram: ComponentType<DisruptionDiagramData> = (props) => {
   );
   const { 0: beginning, [slots.length - 1]: end, ...middle } = slots;
```

Comment:
Ok! I didn't think about this section much, but I do think your version is more readable 👍

Reply:
Yeah sorry, I ended up leaving comments on some lines unchanged by this branch due to reviewing the file as a net-new thing rather than by the diff. Thanks for considering the extra suggestions!